.gitignore

```diff
@@ -2,3 +2,5 @@
 Gemfile.lock
 .bundle
 vendor
+.idea
+lib/com
```

Comment:
once the change to `JARS_HOME` is made, we can go ahead and remove this line from this file